docs/data-sources/events.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Events"
+description: |-
+Get the information for all events from Wavefront
+---
+
+# Data Source: wavefront_events
+
+Use this data source to get information for all events from Wavefront.
+
+## Example Usage
+
+```hcl
+# Get the info for all derived metrics
+data "wavefront_events" "example" {
+}
+```
+
+## Attribute Reference
+
+* `events` - List of all events in Wavefront.
```

Comment:
List of all events in Wavefront. For each event you will see a list of attributes.